services/application.js

```diff
@@ -135,7 +141,7 @@ class Application {
             $set: {reviewComment: document.comment, wholeProgram: document.wholeProgram, status: APPROVED, updatedAt: history.dateTime},
             $push: {history}
         });
-        return (updated.modifiedCount && updated.modifiedCount > 0) ? await this.dbService.find(APPLICATION, {_id: document._id}) : null;
+        return updated.modifiedCount && updated.modifiedCount > 0 ? this.getApplicationById(document._id) : null;
```

Comment:
Can we make this null safe by changing to "updated?.modifiedCount"? "updated" should never be null but we should use null safe checks wherever possible

Reply:
updated